src/OpenTelemetry.ClrProfiler.Native/startup_hook.h

```diff
@@ -0,0 +1,29 @@
+#ifndef DD_CLR_PROFILER_STARTUP_HOOK_H_
+#define DD_CLR_PROFILER_STARTUP_HOOK_H_
+
+#include "string.h"  // NOLINT
+#include "pal.h"
+
+namespace trace
+{
+
+inline WSTRING GetExpectedStartupHookPath(WSTRING home_path) {
+    if (home_path.back() != DIR_SEPARATOR) {
+        home_path.push_back(DIR_SEPARATOR);
+    }
+
+    return home_path + WStr("netcoreapp3.1") + DIR_SEPARATOR +
+            WStr("OpenTelemetry.Instrumentation.StartupHook.dll");
```

Comment:
Not for this PR but we should consider if we want to make this configurable. It makes comparing different versions easy and also allow further customization by the users.

Reply:
Created issue #346 to track this idea - let's discuss at next SIG meeting.